snuba/settings/__init__.py

```diff
@@ -28,6 +28,18 @@
 # Clickhouse Options
 CLICKHOUSE_MAX_POOL_SIZE = 25
 
+SENTRY_LOGICAL_PARTITIONS = 256
+# The number of physical slices that we will need to map resources to
+# for dataset partitioning
+LOCAL_PHYSICAL_SLICES = 1
+# Mapping of logical (key) to physical (value) partitions for datasets
+# that are partitioned across Sentry
+DATASET_PARTITION_MAPPING: Mapping[str, int] = {
+    str(x): 0 for x in range(0, SENTRY_LOGICAL_PARTITIONS)
+}
+# Storage names to apply dataset partitioning to
```

Comment:
Why do datasets need to be partitioned as well? Shouldn't partitioning be defined at the storage layer (i.e. closer to the physical layer)

Reply:
This is just imprecise wording, I can remove the "dataset" wording